src/components/swap/SwapModalFooter.tsx

```diff
@@ -121,46 +175,135 @@ export default function SwapModalFooter({
+    const details: Array<DetailItem> = [
+      { label: t`Exchange rate`, value: `${'1 ' + labelInverted + ' = ' + formattedPrice ?? '-'} ${label}` },
+      {
+        label: t`Network fee`,
+        value: `~${formatCurrencyAmount(trade?.gasUseEstimateUSD, NumberType.FiatGasPrice)}`,
+        labelTooltipText: t`The fee paid to miners who process your transaction. This must be paid in ${nativeCurrency.symbol}.`,
+      },
+      {
+        label: t`Price impact`,
+        value: trade?.priceImpact ? formatPriceImpact(trade?.priceImpact) : '-',
+        color: trade ? getPriceImpactWarning(trade?.priceImpact) : undefined,
+        labelTooltipText: t`The impact your trade has on the market price of this pool.`,
+      },
+    ]
+    if (trade?.tradeType === TradeType.EXACT_INPUT) {
+      details.push({
+        label: t`Minimum received`,
+        value: `${trade.minimumAmountOut(allowedSlippage).toSignificant(6)} ${trade.outputAmount.currency.symbol}`,
+        labelTooltipText: t`The minimum amount you are guaranteed to receive. If the price slips any further, your transaction will revert.`,
+      })
+    } else {
+      details.push({
+        label: t`Maximum sent`,
+        value: `${trade?.maximumAmountIn(allowedSlippage).toSignificant(6)} ${trade?.inputAmount.currency.symbol}`,
+        labelTooltipText: t`The maximum amount you are guaranteed to spend. If the price slips any further, your transaction will revert.`,
+      })
+    }
+
+    return details
+  }, [allowedSlippage, nativeCurrency.symbol, trade])
 
   return (
     <>
-      <AutoRow>
-        <TraceEvent
-          events={[BrowserEvent.onClick]}
-          element={InterfaceElementName.CONFIRM_SWAP_BUTTON}
-          name={SwapEventName.SWAP_SUBMITTED_BUTTON_CLICKED}
-          properties={formatAnalyticsEventProperties({
-            trade,
-            hash,
-            allowedSlippage,
-            transactionDeadlineSecondsSinceEpoch,
-            isAutoSlippage,
-            isAutoRouterApi: routerPreference === RouterPreference.AUTO || routerPreference === RouterPreference.API,
-            swapQuoteReceivedDate,
-            routes,
-            fiatValueInput: fiatValueInput.data,
-            fiatValueOutput: fiatValueOutput.data,
-          })}
-        >
-          <ButtonError
-            data-testid="confirm-swap-button"
-            onClick={onConfirm}
-            disabled={disabledConfirm}
-            style={{ margin: '10px 0 0 0' }}
-            id={InterfaceElementName.CONFIRM_SWAP_BUTTON}
+      <DetailsContainer gap="md">
+        {details.map(({ label, value, color, labelTooltipText }) => (
+          <SwapModalDetailRow
+            key={label}
+            label={label}
+            value={value}
+            color={color}
+            labelTooltipText={labelTooltipText}
+          />
+        ))}
+      </DetailsContainer>
+      {showAcceptChanges ? (
+        <SwapShowAcceptChanges justify="flex-start" data-testid="show-accept-changes">
```

Comment:
nit: Isn't `flex-start` the default? Also, `SwapShowAcceptChanges` is a styled component, so you can just write it as a CSS rule, instead of using a prop. It's a bit more idiomatic in my opinion.